### PR TITLE
CC-25632: Updated builder for Node v18

### PR DESCRIPTION
**Pull request by andrew-maslov**
Ticket: https://spryker.atlassian.net/browse/CC-25632

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "5"
+  - "16"
```
Comment (FRITS-Kh) [inline .travis.yml]: Please replace to
```
dist: jammy
language: node_js
node_js: 18
cache:
    npm: true
```

File: dist/css/query-builder.dark.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*!
- * jQuery QueryBuilder 2.3.3
- * Copyright 2014-2016 Damien "Mistic" Sorel (http://www.strangeplanet.fr)
+ * jQuery QueryBuilder 2.5.2-spryker.1
+ * Copyright 2014-2023 Damien "Mistic" Sorel (http://www.strangeplanet.fr)
```
Comment (FRITS-Kh) [inline dist/css/query-builder.dark.css:3]: Why do we change the license info?

File: dist/js/query-builder.standalone.js [hunk 3]
```diff
@@ -1,137 +1,144 @@
 /*!
- * jQuery.extendext 0.1.1
+ * jQuery.extendext 0.1.2
  *
- * Copyright 2014 Damien "Mistic" Sorel (http://www.strangeplanet.fr)
+ * Copyright 2014-2016 Damien "Mistic" Sorel (http://www.strangeplanet.fr)
```
Comment (FRITS-Kh) [inline dist/js/query-builder.standalone.js:4]: Maybe it should be:
```suggestion
 * Copyright 2014-2023 Damien "Mistic" Sorel (http://www.strangeplanet.fr)
```

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

**Comment by andrew-maslov** [comment]
Closed PR since the repository will be deprecated.

**Pull request closed without merging**